cmd/subspace/main.go

```diff
@@ -122,6 +158,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Default value of endpointHost is the same as httpHost
+	if endpointHost == "" {
+		endpointHost = httpHost
+	}
+
```

Comment:
can you update the cli commands so that the default value for endpointHost is `httpHost` as seen in a different comment. This way this check can be removed (`entrypoint.sh` will just have to set it as a default)

Reply:
I did so, but I got this result.
Please see a screen shot:
<img width="868" alt="スクリーンショット 2020-07-06 15 55 18" src="https://user-images.githubusercontent.com/51456946/86570283-a3b33e00-bfaa-11ea-884c-31f8e440204e.png">

Reply:
prehaps have the default point to `&httpHost` (points to the memory address instead of the current value as `flags` hasn't parsed the values until its parsed them)

If that doesn't work then i'm happy to leave as is

Reply:
Sorry to be late! (To tell you the truth, I was on a vacation!)

> prehaps have the default point to &httpHost (points to the memory address instead of the current value as flags hasn't parsed the values until its parsed them)

I think so too.

OK, I will leave it as is, thanks.